app/containers/message/Video.tsx

```diff
@@ -39,15 +46,34 @@ interface IMessageVideo {
 const Video = React.memo(
 	({ file, showAttachment, getCustomEmoji, theme }: IMessageVideo) => {
 		const { baseUrl, user } = useContext(MessageContext);
+		const [loading, setLoading] = useState(false);
+
 		if (!baseUrl) {
 			return null;
 		}
-		const onPress = () => {
+		const onPress = async () => {
 			if (isTypeSupported(file.video_type)) {
 				return showAttachment(file);
 			}
-			const uri = formatAttachmentUrl(file.video_url, user.id, user.token, baseUrl);
-			openLink(uri, theme);
+
+			if (!isIOS) {
+				const uri = formatAttachmentUrl(file.video_url, user.id, user.token, baseUrl);
+				await downloadVideo(uri);
```

Comment:
In this case I didn't want to redirect the user, because is not all android has a default app to reproduce .WEB videos, so as we cant insurence that, I preferred just download the file to downloads folder.